apps/condo/domains/resident/schema/Resident.js

```diff
@@ -61,6 +79,23 @@ const Resident = new GQLListSchema('Resident', {
             },
         },
 
+        // The reason for this field is to avoid adding check for resident user into global Property read access.
+        // This field have specific use case for mobile client.
+        propertyForResident: {
```

Comment:
residentProperty?

Reply:
Better to use meaningful word first, it's easier to read GraphQL query fields:

```
propertyForResident { ... }
organizationForResident { ... }
```

Reply:
Ah, we have GraphQL-types `ResidentProperty`, — `residentProperty` is better in this case.